physionet-django/project/views.py

```diff
@@ -51,13 +51,14 @@
     UploadedDocument,
 )
 from project.projectfiles import ProjectFiles
-from project.validators import validate_filename
+from project.validators import validate_filename, validate_subdir
 from user.forms import AssociatedEmailChoiceForm
 from user.models import CloudInformation, CredentialApplication, LegacyCredential, User, Training
 
 from django.db.models import F, DateTimeField, ExpressionWrapper
 
 LOGGER = logging.getLogger(__name__)
+MAX_FILENAME_LENGTH_GCP_BUCKET = 256
```

Comment:
This name is misleading - it's not related to a GCS bucket, but an object inside a GCS bucket.

Now that I think of it, we should opt for a new validator function here. For example `validate_gcs_object_name` or `validate_bucket_object_name`. I think it's clearer this way, since now we also have to check for the length. What do you think?

Reply:
I think thats a good idea. We can have gcs related validation over there and we can just add more logic if we need to update the validation in future.

Reply:
Exactly 👍